### DISCUSSION TITLE
Sensitive Data

### DISCUSSION THREAD

**Discussion by tvaughan73** [spatie/laravel-activitylog#1231] [Q&A] [answered]
How should sensitive data be handled when using model events.  I know I can set a field to be excluded but then I would never know a value has changed.  Even if I don't know what the old/new value is I would like to at least know that it changed.  
Is there any way to set some values as masked?

**Comment by lrljoe** [comment, marked as answer]
https://spatie.be/docs/laravel-activitylog/v4/advanced-usage/manipulate-changes-array

This will allow you to mask data
